docs/getting-started/node-nvm.mdx

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 2
+---
+
+# NVM and Node.Js
+
+## Uninstall Node.Js(if you have it installed outside of nvm)
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
+
+
+## Installing the NVM 
+
+Head over to the [NVM Setup](https://www.freecodecamp.org/news/nvm-for-windows-how-to-download-and-install-node-version-manager-in-windows-10/) and follow all the guidelines and intructions
+
+
+:::Note
+To use NVM, you need to open PowerShell or Command Prompt as an admin. You can also use Git bash.
+:::
+
+## Install Node using NVM
+
+To install the latest version of Node, run: 
+```
+nvm install latest
+```
+
+You can also install different versions of Node:
+```
+nvm install node-version-number
+```
```

Comment:
Consider providing examples of how to specify the node version number (eg. `14`, `14.2` as opposed to `v14`).

Reply:
Updated! kindly help me to check again :)